Accepted/asc042.md

```diff
@@ -8,23 +8,22 @@ Donald Dalton <donald@transcriptic.com>
 We should allow creation of `Refs` with initial cover states.
 
 #### **Proposal**
-Allow `Refs` to optionally specify an initial `cover` state from the set of cover types compatible with the corresponding container.
+Allow `Refs` to optionally specify an initial `cover` type from the set of cover types compatible with the corresponding container.
```

Comment:
Given the possible broad interpretation of the word "cover", it'll be good to further clarify and make clear that this field refers to
- the cover-type which a container is initially covered with

And not:
1) a compatible cover-type which the container could be covered with
2) whether the container is covered or not (i.e. cover-state)

Reply:
Yeah this makes sense. My latest changes should make things less ambiguous.